tests/autopilot_pool_expand/aut_pool_expand_test.go

```diff
@@ -0,0 +1,247 @@
+package tests
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	apapi "github.com/libopenstorage/autopilot-api/pkg/apis/autopilot/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	portworx "github.com/portworx/torpedo/drivers/volume/portworx"
+	. "github.com/portworx/torpedo/tests"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	testNameSuite            = "AutPoolResize"
+	timeout                  = 30 * time.Minute
+	scaleTimeout             = 2 * time.Hour
+	workloadTimeout          = 5 * time.Hour
+	retryInterval            = 30 * time.Second
+	unscheduledResizeTimeout = 10 * time.Minute
+)
+
+// Resize storage pool by adding disks by 50% until storage available capacity is less than %70
+var ruleResizeBy50IfPoolAvailableUsageMoreThan70AddDisk = apapi.AutopilotRule{
+	ObjectMeta: meta_v1.ObjectMeta{
+		Name: "storage-pool-resize-add-disk-70",
```

Comment:
for this test, your object name and the label in selector is identical.

But this is not the case for the test below (ruleResizeBy50IfPoolAvailableUsageMoreThan70ResizeDisk)